src/iop/clipping.c

```diff
@@ -47,14 +47,13 @@ DT_MODULE_INTROSPECTION(5, dt_iop_clipping_params_t)
 
 #define CLAMPF(a, mn, mx) ((a) < (mn) ? (mn) : ((a) > (mx) ? (mx) : (a)))
 
-// number of gui guides in combo box
-#define NUM_GUIDES 8
-
 /** flip H/V, rotate an image, then clip the buffer. */
 typedef enum dt_iop_clipping_flags_t
 {
-  FLAG_FLIP_HORIZONTAL = 1,
-  FLAG_FLIP_VERTICAL = 2
+  FLAG_FLIP_NONE = 0,
+  FLAG_FLIP_HORIZONTAL = 1<<0,
+  FLAG_FLIP_VERTICAL = 2<<1,
```

Comment:
Indeed thanks for catching this. I'll fix it this evening
Édouard Gomez
On 31 May 2015 16:04, houz <notifications@github.com> wrote:In src/iop/clipping.c:
>  /** flip H/V, rotate an image, then clip the buffer. */
>  typedef enum dt_iop_clipping_flags_t
>  {
> -  FLAG_FLIP_HORIZONTAL = 1,
> -  FLAG_FLIP_VERTICAL = 2
> +  FLAG_FLIP_NONE = 0,
> +  FLAG_FLIP_HORIZONTAL = 1<<0,
> +  FLAG_FLIP_VERTICAL = 2<<1,

Don't you mean 1<<1?

—Reply to this email directly or view it on GitHub.